src/yara.cc

```
@@ -786,8 +771,6 @@ class AsyncScan : public Nan::AsyncWorker {
 		} catch(std::exception& error) {
 			SetErrorMessage(error.what());
 		}
```

Same, do we want to catch all exceptions?